### ISSUE TITLE
should we show "(slope undefined)" for level 4 challenge?

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/graphing-slope-intercept#7]
According to https://github.com/phetsims/graphing-slope-intercept/issues/2, Level 4, challenge 6 is a Place-the-Points challenge with slope=undefined, and random x-intercept in the range [-10,10] (zero excluded).  

A requirement of Graphing Lines was to put "(slope undefined)" after any equation with an undefined slope.  I'm unclear on whether that's desirable in this situation.

Here's an example of what this challenge looks like:

![screenshot_186](https://cloud.githubusercontent.com/assets/3046552/23812570/4e0cd2ce-0598-11e7-8d42-3d25e0a71cbb.png)
 
If answered correctly, it looks like this:

![screenshot_187](https://cloud.githubusercontent.com/assets/3046552/23812576/5629d006-0598-11e7-8ad0-6d345996374f.png)

If answered incorrectly, it looks like this:

![screenshot_188](https://cloud.githubusercontent.com/assets/3046552/23812586/5e2a959c-0598-11e7-8b55-b00564639eb6.png)

Questions for @amanda-phet:
• Is showing "(slope undefined)" OK, or should it be hidden?
• If hidden, when should it be hidden? Always? (easiest) Before the question is answered?

**Comment by pixelzoom** [comment]
To test-drive this:

• Run [1.0.0-dev.1](http://www.colorado.edu/physics/phet/dev/html/graphing-slope-intercept/1.0.0-dev.1/graphing-slope-intercept_en.html?showAnswers&gameDebug&showReward&shuffleChallenges=false)
• Go to Line Game screen
• Select level 4
• Press the "Skip" button 5 times

**Comment by amanda-phet** [comment]
I would definitely prefer to not have "(slope undefined)" shown in the challenge prompt. It would be nice to show it in the "your line" box, but if that is complicated I'm fine with it never showing up in a challenge. (It should still show up on the explore screen, though!).

**Comment by pixelzoom** [comment]
@amanda-phet If I can make it appear in the "Your Line" box, are you OK with it never appearing in the "Line to Graph" box?

**Comment by pixelzoom** [comment]
Prior to challenge 6 in level 4 (Place-the-Points with undefined slope and random non-zero y-intercept), challenges with undefined slopes were not allowed.  So all I had to do for this issue was to remove the "(slope undefined)" in the "Line to Graph" box.   That involved touching more Graphing Lines code than I would have liked (see above commit), but I proceeded with caution and did a fuzzMouse test to verify that I didn't break anything.  

@amanda-phet here's a dev version to verify:

http://www.colorado.edu/physics/phet/dev/html/graphing-slope-intercept/1.0.0-dev.3/graphing-slope-intercept_en.html?showAnswers&gameDebug&shuffleChallenges=false

Here's what the initial challenge looks like:

![screenshot_191](https://cloud.githubusercontent.com/assets/3046552/23837155/82639596-0749-11e7-8a3b-16e6a869772c.png)

Here's what a correct answer looks like:

![screenshot_193](https://cloud.githubusercontent.com/assets/3046552/23837157/8a31cbbc-0749-11e7-89cf-68b9a8eb0f5b.png)

Here's what an incorrect answer looks like:

![screenshot_192](https://cloud.githubusercontent.com/assets/3046552/23837159/90fc080e-0749-11e7-8d69-e3d4252be2f7.png)
